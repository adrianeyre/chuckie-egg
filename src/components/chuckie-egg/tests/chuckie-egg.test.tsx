import { render } from '@testing-library/react';

import ChuckieEgg from '../chuckie-egg';
import IChuckieEggProps from '../interfaces/chuckie-egg-props';

describe('Battle Ships', () => {
	it('Should render correctly', () => {
		const defaultProps: IChuckieEggProps = {};
		const chuckieEgg = render(<ChuckieEgg {...defaultProps} />);
		expect(chuckieEgg).toMatchSnapshot();
	});
});