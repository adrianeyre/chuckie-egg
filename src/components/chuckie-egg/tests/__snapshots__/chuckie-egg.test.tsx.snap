// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Battle Ships Should render correctly 1`] = `ShallowWrapper {}`;
